internal/config/config.go

```diff
@@ -377,24 +585,56 @@ var regionalConfig = map[string]RegionalConfig{
 			"2060": {
 				SKU: strPtr("5394903600"),
 			},
-			"3070": {},
+			"2070": {
+				SKU: strPtr("5394902600"),
+			},
+			"2080": {
+				SKU: strPtr("5336531700"),
+			},
+			"2080TI": {
+				SKU: strPtr("5218988100"),
+			},
+			"TITAN-RTX": {
+				SKU: strPtr("5256301700"),
+			},
+			"TITAN-V": {
+				SKU: strPtr("5145366000"),
+			},
+			"3070": {
+			},
 			"3080": {
 				SKU: strPtr("5438797200"),
 			},
 			"3090": {
 				SKU: strPtr("5438797100"),
 			},
 		},
-		Locale:       "no_NO",
-		NvidiaLocale: "no-NO",
+		Locale:       "no_no",
+		NvidiaLocale: "no-no",
 		Currency:     "NOK",
 	},
 	"NLD": {
 		Models: map[string]Model{
 			"2060": {
 				SKU: strPtr("5394903500"),
 			},
-			"3070": {},
+			"2070": {
+				SKU: strPtr("5394902200"),
+			},
+			"2080": {
+				SKU: strPtr("5336532100"),
+			},
+			"2080TI": {
+				SKU: strPtr("5218614400"),
+			},
+			"TITAN-RTX": {
```

Comment:
So if every region as the 2060/2070/2080/2080TI lets just keep it to those, TITAN-RTX/TITAN-V seem like there only in some regions and I think that's rather confusing for folks.

Reply:
ok

Reply:
can we leave it but not document it so if you change your mind we could use it?